Modules/vc-module-inventory/VirtoCommerce.InventoryModule.Web/Module.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using VirtoCommerce.Domain.Search;
+using VirtoCommerce.InventoryModule.Core.Services;
+using VirtoCommerce.InventoryModule.Data.Repositories;
+using VirtoCommerce.InventoryModule.Data.Search.Indexing;
+using VirtoCommerce.InventoryModule.Data.Services;
+using VirtoCommerce.InventoryModule.Web.JsonConverters;
+using VirtoCommerce.Platform.Core.Modularity;
+
+namespace VirtoCommerce.InventoryModule.Web
+{
+    public class Module : IModule
+    {
+        public ManifestModuleInfo ModuleInfo { get; set; }
+        public void Initialize(IServiceCollection serviceCollection)
+        {
+            var configuration = serviceCollection.BuildServiceProvider().GetRequiredService<IConfiguration>();
+            serviceCollection.AddTransient<IInventoryRepository, InventoryRepositoryImpl>();
+            serviceCollection.AddDbContext<InventoryDbContext>(options => options.UseSqlServer(configuration.GetConnectionString("VirtoCommerce")));
```

Comment:
Should be used own inventory module connection string and  `VirtoCommerce` should be used only as default.
See the example https://github.com/VirtoCommerce/vc-module-pricing/blob/master/VirtoCommerce.PricingModule.Web/Module.cs#L27